includes/course-theme/class-sensei-course-theme.php

```diff
@@ -145,20 +147,51 @@ public function override_theme() {
 	}
 
 	/**
-	 * Add a route for loading the course theme.
+	 * Add learning mode prefix as query var.
 	 *
 	 * @access private
 	 */
-	public function add_rewrite_rules() {
+	public function add_query_var() {
 		global $wp;
+
 		$wp->add_query_var( self::QUERY_VAR );
-		add_rewrite_rule( '^' . self::QUERY_VAR . '/([^/]*)/([^/]*)/?\??(.*)', 'index.php?' . self::QUERY_VAR . '=1&post_type=$matches[1]&name=$matches[2]&$matches[3]', 'top' );
-		add_rewrite_tag( '%' . self::QUERY_VAR . '%', '([^?]+)' );
+	}
+
+
+	/**
+	 * Flush rewrite rules if changed.
+	 *
+	 * @access private
+	 */
+	public function maybe_flush_rewrite_rules() {
+
+		delete_option( 'sensei_course_theme_query_var_flushed' );
```

Comment:
Probably we forgot to remove this line from a test, right?

Also, I noticed we forgot to add it to the `Sensei_Data_Cleaner` to be removed on uninstall.

Reply:
Oh also 7466c2c to not update it always.